local/mix/version_upgrade_test.ex

```diff
@@ -68,10 +77,15 @@ defmodule Mix.Tasks.AntikytheraLocal.VersionUpgradeTest do
   end
 
   defp version(app_name) do
-    case rpc(AntikytheraCore.Version, :current_version, [app_name]) do
-      {:badrpc, reason} -> raise "Failed to communicate with the running node! reason: #{reason}"
-      value             -> value
-    end
+    rpc!(AntikytheraCore.Version, :current_version, [app_name])
+  end
+
+  defp module_md5(app_name) do
+    mod_name = app_name
+      |> Atom.to_string
+      |> Macro.camelize()
+    mod = Module.concat([Elixir, mod_name])
```

Comment:
```suggestion
    mod = Module.safe_concat(Elixir, mod_name)
```

Reply:
During the test, test process and web server process exist. `mod_name` is either `:antikythera_instance_example` or `:testgear`.
Web server process refers both `AntikytheraInstanceExample` and `Testgear` but perhaps test process refers `AntikytheraInstanceExample` only. I thought that using `Module. safe_concat` may fail.

Reply:
Yay, test passed.